<launch>

    <arg name="path" default="$(find final_simulator)/csv" />
    <arg name="ns" default="front" />

    <arg name="refPath" default="$(find final_evaluation)/csv" />
    <arg name="laneId" default="1" />

    <node pkg="final_evaluation" type="evaluation" name="evaluation" output="screen">
        <param name="refPath" value="$(arg refPath)" type="str"/>
        <param name="laneId" value="$(arg laneId)" type="str"/>
    </node>

    <group ns="$(arg ns)">
        <node pkg="final_evaluation" type="master_vehicle" name="master_vehicle">
            <param name="ns" value="$(arg ns)"/>
            <param name="init_x" value="30.0" />
            <param name="init_y" value="1.65" />
            <param name="init_yaw" value="0.0" />
            <param name="refPath" value="$(arg refPath)" type="str"/>
            <param name="laneId" value="$(arg laneId)" type="str"/>
        </node>
        <node pkg="final_simulator" type="display" name="display"/>
    </group>
</launch>
